CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 6.4.11
+### Added
+- Added a new function to `IterableAPI` to get the currently set authentication token
+
+### Changed
+- Changed `setEmail` and `setUserId` to run through the login process if also sent in an authentication token (and a non-`nil` ID)
+- Changed authentication manager behavior to schedule automatic refreshes on JWT retrieval or decoding failures
+
```

Comment:
Fixes instances where setting a new authToken for same user failed

Reply:
OOPS I like this too